Index: PropellerCompiler/SymbolEngine.cpp
===================================================================
--- PropellerCompiler/SymbolEngine.cpp	(revision 69)
+++ PropellerCompiler/SymbolEngine.cpp	(working copy)
@@ -16,6 +16,8 @@
 #include "Utilities.h"
 #include <string.h>
 
+#define stricmp strcasecmp
+
 static SymbolTableEntryDataTable symbols[] =
 {
     {type_left,             0,                  "(",            0,                  false}, //miscellaneous
Index: SpinSource/objectheap.cpp
===================================================================
--- SpinSource/objectheap.cpp	(revision 69)
+++ SpinSource/objectheap.cpp	(working copy)
@@ -14,6 +14,8 @@
 #include "../PropellerCompiler/PropellerCompiler.h"
 #include "objectheap.h"
 
+#define stricmp strcasecmp
+
 // Object heap (compile-time objects)
 struct ObjHeap
 {
